registry/storage/driver/middleware/cloudfront/middleware.go

```diff
@@ -100,11 +106,21 @@ func newCloudFrontStorageMiddleware(storageDriver storagedriver.StorageDriver, o
 		}
 	}
 
+	var acceptedRegions []string
```

Comment:
It would be good to add this to the configuration docs so we understand its intended use, along with an example.